File: 4) C#/troco.cs
using System;

namespace Troco
{
	class Program
	{
		static void Main( string[ ] args )
		{
			float troco, preco, pagamento;
			int qtd;
			
			Console.WriteLine( "Digite a quantidade de produto: " );
			qtd = Convert.ToInt32( Console.ReadLine( ) );
			
			Console.WriteLine( "Digite o preco do produto: " );
			preco = float.Parse( Console.ReadLine( ) );
			
			Console.WriteLine( "Digite o dinheiro pago: " );
			pagamento = float.Parse( Console.ReadLine( ) );
			
			troco = pagamento - ( preco * qtd );
			Console.WriteLine( "Troco : " + troco );
		}
	}
}	

File: 6) C E C++/troco.c
#include <stdio.h>

int main( ) {
	int quantidade;
	float preco, pagamento, troco;
	
	printf( "Digite a quantidade de produto comprado: " );
	scanf( "%d", &quantidade );
	printf( "Digite o preco do produto: " );
	scanf( "%f", &preco );
	printf( "Digite o dinheiro pago: " );
	scanf( "%f", &pagamento );
	
	troco = pagamento - ( quantidade * preco );
	
	printf( "Troco: %.2f", troco );
}